src/js/controllers/ctrl-media-import.js

```diff
@@ -52,7 +56,12 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 	let _allFiles = []
 
 	const onMediaSelect = (media) => {
-		$window.parent.Materia.Creator.onMediaImportComplete([media])
+		setTimeout(() => {
```

Comment:
Is this necessary?

If this is being used to give an asynchronous action time to complete, it should be a callback of that asynchronous action instead. If this is being used to create the illusion of something taking a moment to complete, 10 milliseconds is so small that you may as well not bother.